### PR TITLE
Feature/add area specs

### PR DESCRIPTION
**Pull request by DerekMaffett**
Adds area specs and finishes a refactor started here: https://github.com/FormidableLabs/victory-core/pull/211

### CODE REVIEW

File: src/victory-primitives/utils.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export default {
```
Comment (boygirl) [inline src/victory-primitives/utils.js]: This method can go in `victory-util/collection`.

**Review by boygirl — CHANGES_REQUESTED** [review]
Approved pending moving splitArray method to `victory-util/collection`

File: src/victory-primitives/utils.js (hunk 1 above)
Comment (DerekMaffett) [reply src/victory-primitives/utils.js]: @boygirl Done

**Review by boygirl — APPROVED** [review]
Approved!

**Pull request merged**